src/pages/AccountPage.tsx

```diff
@@ -0,0 +1,82 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { Container, Card, Badge } from 'react-bootstrap'
+import { useNavigate } from 'react-router-dom'
+import useTranslationHook from '../hook/TranslationHook'
+
+const AccountPage = (): JSX.Element => {
+  const navigate = useNavigate()
+  const { t } = useTranslationHook()
+
+  const goToProfile = (): void => {
+    navigate('account/profile')
```

Comment:
Broken link, you're probably missing the leading slash

Reply:
yes resolved